schema.cc

```diff
@@ -807,12 +807,23 @@ std::ostream& schema::describe(replica::database& db, std::ostream& os) const {
             os << "\n    WHERE " << view_info()->where_clause();
         }
     } else {
-        os << " TABLE " << cql3::util::maybe_quote(ks_name()) << "." << cql3::util::maybe_quote(cf_name()) << " (";
+        os << "TABLE " << cql3::util::maybe_quote(ks_name()) << "." << cql3::util::maybe_quote(cf_name()) << " (";
         for (auto& cdef : all_columns()) {
+            if (with_internals && dropped_columns().contains(cdef.name_as_cql_string())) {
+                continue;
+            }
+
             os << "\n    ";
             column_definition_as_cql_key(os, cdef);
             os << ",";
         }
+
+        if (with_internals) {
+            for (auto& cdef: dropped_columns()) {
+                os << "\n    ";
+                os << cdef.first << " " << cdef.second.type->cql3_type_name() << ",";
```

Comment:
By the way, every review comment here that turns out right is a good potential for adding a test which checks exactly that case - and fails with the broken code and works with the fixed code.

Reply:
According to Cassandra if column was dropped and later readded, the "readded" column is skipped and the dropped one is printed to avoid doubling it.

Also after the table description, there are `ALTER TABLE ... DROP ...` and `ALTER TABLE ... ADD ...` statements to show recreation of the column.

I've left a comment there.

Reply:
Oh, but there is a problem if a added table have the same name but different types. Should I print both present and dropped column then(with the same name)?
I'll add a test for this and other situations like this, too.